dolphinscheduler-task-plugin/dolphinscheduler-task-sqoop/pom.xml

```diff
@@ -48,5 +48,10 @@
             <artifactId>dolphinscheduler-datasource-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.dolphinscheduler</groupId>
+            <artifactId>dolphinscheduler-service</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
I am not sure wether we should dependent `dolphinscheduler-service` in task plugin or not

Reply:
Do you have time to take a look @caishunfeng @ruanwenjun

Reply:
> I am not sure wether we should dependent `dolphinscheduler-service` in task plugin or not

We should not add the service module in task-plugin, they are independent.

Reply:
> I am not sure wether we should dependent `dolphinscheduler-service` in task plugin or not
> Do you have time to take a look @caishunfeng @ruanwenjun

Hi @zhongjiajie Thanks a lot for your review and comment.

I add this dependent since `sqoop` task plugin needs to add its own mask pattern to `SensitiveDataConverter`.

Maybe we should discuss whether it is appropriate to do so.
cc @caishunfeng @ruanwenjun